Sources/Utility/Process.swift

```diff
@@ -151,6 +155,9 @@ public final class Process: ObjectIdentifierProtocol {
     /// Queue to protect concurrent reads.
     private let serialQueue = DispatchQueue(label: "org.swift.swiftpm.process")
 
+    /// cache of validated executables arguments
```

Comment:
Please use proper capitalization, punctuation in all comments. http://llvm.org/docs/CodingStandards.html#commenting